lib/client-assets.php

```diff
@@ -111,6 +111,10 @@ function register_tinymce_scripts() {
 function gutenberg_override_script( $handle, $src, $deps = array(), $ver = false, $in_footer = false ) {
 	wp_deregister_script( $handle );
 	wp_register_script( $handle, $src, $deps, $ver, $in_footer );
+
+	if ( function_exists( 'wp_set_script_translations' ) ) {
+		wp_set_script_translations( $handle, 'gutenberg' );
```

Comment:
Commenting this line resolves the memory exhaustion for me. Perhaps getting stuck in some infinite recursion where `wp_set_script_translations` is in-turn causing `gutenberg_override_script` to be called once more?